docs/upgrade.html

```diff
@@ -70,6 +70,10 @@ <h5><a id="upgrade_120_notable" href="#upgrade_120_notable">Notable changes in 1
     <li><a href="https://issues.apache.org/jira/browse/KAFKA-5674">KAFKA-5674</a> extends the lower interval of <code>max.connections.per.ip minimum</code> to zero and therefore allows IP-based filtering of inbound connections.</li>
     <li> New Kafka Streams configuration parameter <code>upgrade.from</code> added that allows rolling bounce upgrade from older version. </li>
 </ul>
+<ul>
+    <li><a href="https://cwiki.apache.org/confluence/x/DVyHB">KIP-284</a> changed the retention time for repartition topic by setting its default value to Long.MAX_VALUE.</li>
+    <li>Use purge data API to keep storage small</li>
```

Comment:
I would omit this sentence here (this html page shown only the highlights -- the details are provided in the KS specific upgrade page.

Reply:
ok